### ISSUE TITLE
Error when starting devserver under Python 3.8 and Windows 10

### ISSUE THREAD

**Issue by JKRhb** [oTree-org/otree-core#509]
I recently tried to use the `otree devserver` command under Windows 10 using Python 3.8 and got the following error messages:

```python
Traceback (most recent call last):
  File "c:\python38\lib\site-packages\otree_startup\__init__.py", line 212, in do_django_setup
    django.setup()
  File "c:\python38\lib\site-packages\django\__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "c:\python38\lib\site-packages\django\apps\registry.py", line 91, in populate
    app_config = AppConfig.create(entry)
  File "c:\python38\lib\site-packages\django\apps\config.py", line 116, in create
    mod = import_module(mod_path)
  File "c:\python38\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "c:\python38\lib\site-packages\channels\apps.py", line 6, in <module>
    import daphne.server
  File "c:\python38\lib\site-packages\daphne\server.py", line 18, in <module>
    asyncioreactor.install()
  File "c:\python38\lib\site-packages\twisted\internet\asyncioreactor.py", line 320, in install
    reactor = AsyncioSelectorReactor(eventloop)
  File "c:\python38\lib\site-packages\twisted\internet\asyncioreactor.py", line 69, in __init__
    super().__init__()
  File "c:\python38\lib\site-packages\twisted\internet\base.py", line 571, in __init__
    self.installWaker()
  File "c:\python38\lib\site-packages\twisted\internet\posixbase.py", line 286, in installWaker
    self.addReader(self.waker)
  File "c:\python38\lib\site-packages\twisted\internet\asyncioreactor.py", line 151, in addReader
    self._asyncioEventloop.add_reader(fd, callWithLogger, reader,
  File "c:\python38\lib\asyncio\events.py", line 501, in add_reader
    raise NotImplementedError
NotImplementedError
```

A quick google research revealed to me that this seems to be a common problem with Windows and Python 3.8. Do you think there is a way to fix this? Or should oTree only be used with Python 3.7? Thanks in advance!

**Comment by oTree-org** [comment]
How did you install it on python 3.8? the oTree installer is supposed to prevent being installed on 3.8.

**Comment by JKRhb** [comment]
> 
> How did you install it on python 3.8? the oTree installer is supposed to prevent being installed on 3.8.

Oh, okay, that explains something. I could install it by simply using `pip3 install -U otree`.

**Comment by oTree-org** [comment]
Oh you are right, I just realised I had not published that change yet. It's published now, so oTree is blocked on Python 3.8 until we get Twisted support.

**Comment by JKRhb** [comment]
Alright, thank you for the clarification! I switched to Python 3.7 in the meantime.
